config/cmake/HDF5_Examples.cmake.in

```diff
@@ -81,15 +81,14 @@ set(ADD_BUILD_OPTIONS "${ADD_BUILD_OPTIONS} -DSITE:STRING=${CTEST_SITE} -DBUILDN
 #endif()
 
 ###############################################################################################################
+set(ADD_BUILD_OPTIONS "${ADD_BUILD_OPTIONS} -DHDF5_ROOT:PATH=${INSTALLDIR}")
 if(WIN32)
   set(SITE_OS_NAME "Windows")
-  set(ENV{HDF5_ROOT} "${INSTALLDIR}")
   set(CTEST_BINARY_NAME ${CTEST_SOURCE_NAME}\\build)
   set(CTEST_SOURCE_DIRECTORY "${CTEST_DASHBOARD_ROOT}\\${CTEST_SOURCE_NAME}")
   set(CTEST_BINARY_DIRECTORY "${CTEST_DASHBOARD_ROOT}\\${CTEST_BINARY_NAME}")
 else()
   set(ENV{HDF5_ROOT} "${INSTALLDIR}")
-  set(ENV{LD_LIBRARY_PATH} "${INSTALLDIR}/lib")
```

Comment:
Did you intend to delete the `HDF5_ROOT` line rather than the `LD_LIBRARY_PATH` line here, or is this correct?

Reply:
Good catch!